Makefile

```diff
@@ -189,6 +190,7 @@ else
 	--build-arg DOCKER_BASE_SUFFIX=$(DOCKER_BASE_SUFFIX) \
 	--build-arg LIBYAML_VERSION=$(LIBYAML_VERSION) \
 	--build-arg EDITION=$(EDITION) \
+	--build-arg ENABLE_KONG_LICENSING=$(ENABLE_KONG_LICENSING) \
```

Comment:
I'd have thought this is the only spot this is required in the `Makefile` no?

Reply:
umm... good point. I guess EDITION is a bit different because it is not only used when building openresty. let me double check this.

Reply:
fixed